modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/TopicMessageDeliveryStrategy.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Since this seems to be a configuration related class, can move to org.wso2.andes.configuration.utils